server/src/main/java/com/testsigma/controller/StorageConfigController.java

```diff
@@ -27,9 +30,14 @@ public StorageConfigDTO get() {
   }
 
   @RequestMapping(method = RequestMethod.PUT)
-  public StorageConfigDTO update(@RequestBody StorageConfigRequest storageConfigRequest) {
+  public ResponseEntity<StorageConfigDTO> update(@RequestBody StorageConfigRequest storageConfigRequest) throws TestsigmaException {
```

Comment:
@shanmukha2000 throw customException and send APIErrorDTO from globalException Handler